### DISCUSSION TITLE
How to access raw html in browser before Nuxt starts to add listeners and deserialize data sent by useState ?

### DISCUSSION THREAD

**Discussion by geminigeek** [nuxt/nuxt#26053] [Questions]
hi,

i want to access raw html (in client / browser) before nuxt starts to add listeners and deserialize data sent by useState, i have checked this page https://nuxt.com/docs/api/advanced/hooks for lifecycle hooks but no hooks seems good for the use case ,

is there a way to achieve this ?

**Comment by TheAlexLichter** [comment]
What do you want to achieve?

You can use a nitro hook when the HTML is sent (`render:html`, `render:response`) or use [custom revivers](https://www.youtube.com/watch?v=8w6ffRBs8a4) to change how things are serialized ☺️

**Reply by geminigeek** [reply]
i am modifying some data in nitro before sending it to browser , i want to run some code to re-modify before Nuxt/Vue use it, only want re-modified data if js is enabled in browser.
